app/presenters/v3/service_plan_presenter.rb

```diff
@@ -90,7 +90,7 @@ def maintenance_info
         end
 
         def parse_schema(schema)
-          return {} unless schema
+          return { parameters: {} } unless schema
 
           { parameters: JSON.parse(schema) }
         rescue JSON::ParserError
```

Comment:
What about if there's a parse error?

Reply:
Moved the "parameter" property to the calling code for `parse_schema`.